### ISSUE TITLE
can you provide the code of cc_clip_logits.pk?

### ISSUE THREAD

**Issue by ZHUXUHAN** [thunlp/VisualDS#8]
hi, do you still have the related code for generating cc_clip_logits.pk? can you help me with providing it?

**Comment by waxnkw** [comment]
Please download from the [link](https://thunlp.oss-cn-qingdao.aliyuncs.com/visds/generate_logits.py). This code is used to calculate the logits with CLIP model.

**Comment by ZHUXUHAN** [comment]
> Please download from the [link](https://thunlp.oss-cn-qingdao.aliyuncs.com/visds/generate_logits.py). This code is used to calculate the logits with CLIP model.

Thank you very much for your supporting.

**Comment by ZHUXUHAN** [comment]
here is the 'l = pickle.load(open("50vgkb_clip_data.pk", "rb"))' in the code, is th
